src/components/DpMultiselect/DpMultiselect.vue

```diff
@@ -152,6 +230,12 @@ export default {
       default: ''
     },
 
+    tagPlaceholder: {
+      type: String,
+      required: false,
+      default: () => Translator.trans('tag.create')
```

Comment:
Not a blocker right now (it is merged anyways) but didn't we want to drop Translator in demosplan-ui?

Reply:
But we still have no concept how to handle things like that. so unfortunately we have to do it this way until we have a technical solution